src/assets/translations/fr/main.json

```diff
@@ -1464,7 +1470,8 @@
       "andMore": "... et plus",
       "snapInstalledToast": "Snap ShapeShift MultiChain pour MetaMask installé",
       "snapUninstalledToast": "Snap ShapeShift MultiChain pour MetaMask désinstallé",
-      "dontAskAgain": "Ne plus demander"
+      "dontAskAgain": "Ne plus demander",
+      "multiAccountUnsupported": "Le multi-compte n'est pas pris en charge pour cette chaîne en utilisant le Multichain Snap avec MetaMask"
```

Comment:
```suggestion
      "multiAccountUnsupported": "Le multi-compte n'est pas pris en charge pour cette chaîne en utilisant le Snap Multichain avec MetaMask"
```

Reply:
Thanks for the review and suggestion, good  call 👍